sorc/link_workflow.sh

```diff
@@ -191,6 +191,7 @@ declare -a ufs_templates=("model_configure.IN" "input_global_nest.nml.IN"
   "ufs.configure.s2swa_esmf.IN"
   "ufs.configure.leapfrog_atm_wav.IN"
   "ufs.configure.leapfrog_atm_wav_esmf.IN"
```

Comment:
the `_esmf.IN` templates have been removed from the ufs-wx-model.  They should not be linked here anymore with the update of the model hash.

Reply:
good catch! fixed!